gridscale/provider.go

```diff
@@ -78,6 +78,7 @@ func Provider() terraform.ResourceProvider {
 			"gridscale_backupschedule":                 resourceGridscaleStorageBackupSchedule(),
 			"gridscale_paas":                           resourceGridscalePaaS(),
 			"gridscale_paas_securityzone":              resourceGridscalePaaSSecurityZone(),
+			"gridscale_postgres":                       resourceGridscalePostgresSQL(),
```

Comment:
Suggestion: We should name the resource `gridscale_postgresql` since PostgreSQL is the product name.